features/demo/src/demo.js

```diff
@@ -28,6 +28,21 @@ window.addEventListener("DOMContentLoaded", () => {
             });
         });
 
+    document
```

Comment:
It could be tested with a mock server. Unit test at the mock implementation level would also be nice (for instance, the AsyncPod). Anyway, it's more a "nice to have".